pysat/instruments/pysat_netcdf.py

```diff
@@ -0,0 +1,248 @@
+        tag unless specified by user at Instrument instantiation. (default='')
+    inst_id : str
+        Satellite ID used to identify particular data set to be loaded.
+        This input is nominally provided by pysat itself. (default='')
+    strict_meta : bool
+        Flag that checks if metadata across fnames is the same if True
+        (default=False)
+    file_format : str
+        file_format keyword passed to netCDF4 routine.  Expects one of
+        'NETCDF3_CLASSIC', 'NETCDF3_64BIT', 'NETCDF4_CLASSIC', or 'NETCDF4'.
+        (default='NETCDF4')
+    epoch_name : str
+        Data key for epoch variable.  The epoch variable is expected to be an
+        array of integer or float values denoting time elapsed from an origin
+        specified by `epoch_origin` with units specified by `epoch_unit`. This
+        epoch variable will be converted to a `DatetimeIndex` for consistency
+        across pysat instruments.  (default='Epoch')
+    epoch_unit : str
+        The pandas-defined unit of the epoch variable ('D', 's', 'ms', 'us',
+        'ns'). (default='ms')
+    epoch_origin : str or timestamp-convertable
+        Origin of epoch calculation, following convention for
+        `pandas.to_datetime`.  Accepts timestamp-convertable objects, as well as
+        two specific strings for commonly used calendars.  These conversions are
+        handled by `pandas.to_datetime`.
+        If ‘unix’ (or POSIX) time; origin is set to 1970-01-01.
+        If ‘julian’, `epoch_unit` must be ‘D’, and origin is set to beginning of
+        Julian Calendar. Julian day number 0 is assigned to the day starting at
+        noon on January 1, 4713 BC. (default='unix')
+    pandas_format : bool
+        Flag specifying if data is stored in a pandas DataFrame (True) or
+        xarray Dataset (False). (default=False)
+    decode_timedelta : bool
+        Used for xarray data (`pandas_format` is False).  If True, variables
+        with unit attributes that  are 'timelike' ('hours', 'minutes', etc) are
+        converted to `np.timedelta64`. (default=False)
+    labels : dict
+        Dict where keys are the label attribute names and the values are tuples
+        that have the label values and value types in that order.
+        (default={'units': ('units', str), 'name': ('long_name', str),
+        'notes': ('notes', str), 'desc': ('desc', str),
+        'min_val': ('value_min', np.float64),
+        'max_val': ('value_max', np.float64), 'fill_val': ('fill', np.float64)})
+
+    Returns
+    -------
+    data : pds.DataFrame or xr.Dataset
+        Data to be assigned to the pysat.Instrument.data object.
+    mdata : pysat.Meta
+        Pysat Meta data for each data variable.
+
+    """
+
+    # netCDF4 files, particularly those produced by pysat can be loaded using a
+    # pysat provided function, load_netcdf4.
+    data, mdata = pysat.utils.load_netcdf4(fnames, strict_meta=strict_meta,
```

Comment:
use the non-deprecated version in `utils.io`

Reply:
Done!